tests/conftest.py

```diff
@@ -573,6 +567,31 @@ def wait_for_camouflage(host="localhost", port=8000, timeout=30):
     return False
 
 
+def wait_for_camouflage(host: str = "localhost", port: int = 8000, timeout: int = 30):
+    url = f"http://{host}:{port}/ping"
+
+    def parse_fn(resp: requests.Response) -> bool:
+        if (resp.json()['message'] == 'I am alive.'):
+            return True
+
+        warnings.warn(("Camoflage returned status 200 but had incorrect response JSON. Continuing to wait. "
+                       "Response JSON:\n%s"),
+                      resp.json())
+        return False
+
+    return wait_for_server(url, timeout=timeout, parse_fn=parse_fn)
+
+
+def wait_for_milvus(host: str = "localhost", port: int = 19530, timeout: int = 180):
+    url = f'http://{host}:{port}/healthz'
+
+    def parse_fn(resp: requests.Response) -> bool:
+        content = resp.content.decode('utf-8')
+        return 'OK' in content
```

Comment:
Can we also check the status code? I would hate for it to return a 400 error with `Im not OK` and this test would pass

Reply:
It doesn't even call that method if the status code isn't 200:
```python
            if (resp.status_code == 200):
                if parse_fn(resp):
                    return True
```